test_libs/pyspec/eth2spec/utils/ssz/ssz_typing.py

```diff
@@ -1,142 +1,171 @@
+from typing import Dict, Iterator
 from types import GeneratorType
-from typing import List, Iterable, TypeVar, Type, NewType
-from typing import Union
-from typing_inspect import get_origin
```

Comment:
Could remove `typing_inspect` from `requirements.txt` and `setup.py`.

Reply:
See d1fa3ac